### PR TITLE
Example changes

### PR DESCRIPTION
**Pull request by tgsoverly**

### CODE REVIEW

File: test-projects/demo/dashboards/welcome.visivo.yml [hunk 1]
```diff
@@ -9,30 +9,37 @@ models:
...
+          # Equivalent to:
+          # color:
+          #   static: #b97a9b
       decreasing:
         marker:
           color: "#edbdb5"
-    order_by:
-      - query(row asc)
-    filters:
-      - query( waterfall is not null )
```
Comment (jpjsonic) [inline test-projects/demo/dashboards/welcome.visivo.yml:33]: I think I actually like requiring the query function here. Even though it's more verbose it makes it really easy to glean what's going into the trace query without knowing much about the attributes of the trace

File: test-projects/demo/dashboards/welcome.visivo.yml [hunk 2]
```diff
@@ -72,9 +79,9 @@ dashboards:
           - width: 2
             table:
               name: Fibonacci Waterfall Table
-              selector: ref(Fibonacci Waterfall Selector)
+              selector: Fibonacci Waterfall Selector
               traces:
-                - ref(Fibonacci Waterfall Trace)
```
Comment (jpjsonic) [inline test-projects/demo/dashboards/welcome.visivo.yml:77]: What's your thinking behind getting rid of refs? I find them to be pretty useful to really quickly see when visivo is referring to another object in a configuration block.

Comment (tgsoverly) [reply test-projects/demo/dashboards/welcome.visivo.yml:77]: My thought is that it isn't needed.  We never have a situation where we can have a string or an object that isn't a ref.

File: test-projects/project.visivo.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
+# V2 Spec
```
Comment (tgsoverly) [inline test-projects/project.visivo.yml:1]: Looking at this I think there are three types of things that we need to be consistent about:
1. Items that reference the project itself. `trace.meta.date-grain`
2. Items that reference the output of queries
3. Items that have executable code

In addition we have emerging need to combined at least 1 and 3 if we do `Selector` where you can have a matcher.

File: test-projects/project.visivo.yml [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  - name: if_production
+    logic: if env == production
+  - name: if_morning_notifications
+    logic: if env["MORNING_NOTIFICATION"] == true
+
+alerts:
+  - name: Failure During Business Hours
+    condition: if_during_business_hours
+    message: |
+      Trigger {{ trigger.name }} has failed
```
Comment (tgsoverly) [inline test-projects/project.visivo.yml:23]: We could always do something like `{{ ... }}` for project references.  But `project(...)` would be more consistent.  The starting point will always need to be understood.  Relative to what point.

Comment (tgsoverly) [reply test-projects/project.visivo.yml:23]: I do like {{ }} because it stand out.

Comment (tgsoverly) [reply test-projects/project.visivo.yml:23]: - logic: assert_that( {{ Aggregated Line.props.marker.line.color[1] }} ).is_equal_to("#D25946")

**Pull request closed without merging**